pages/common/bedtools.md

```diff
@@ -0,0 +1,28 @@
+# bedtools
+
+> A swiss-army knife of tools for genomic-analysis tasks.
+> Used to intersect, group, convert and count data in  BAM, BED, GFF/GTF, VCF format.
+
+- Intersect two files strand-specific and save the result to {{path/to/output_file}}:
```

Comment:
"files strand-specific" - the grammar doesn't feel right with this one. Shouldn't it be strand-specific files ?